src/pages/layer-2.js

```diff
@@ -0,0 +1,1097 @@
+            </p>
+
+            <h3>Scalability</h3>
+            <p>
+              The main goal of scalability is to increase transaction speed
+              (faster finality), and transaction throughput (high transactions
+              per second), without sacrificing decentralization or security
+              (more on the <Link to="/upgrades/vision/  ">Ethereum vision</Link>
+              ).
+            </p>
+            <p>
+              The Ethereum community has taken a strong stance that it would not
+              throw out decentralization or security in order to scale. Until
+              sharding, Ethereum Mainnet (layer 1) will only be able to process{" "}
+              <Link to="https://ethtps.info/Network/Ethereum">
+                roughly 15 transactions per second
+              </Link>
+              . When demand to use Ethereum is high this causes network
+              congestion, increasing transaction fees, and pricing out those who
+              cannot afford it from using Ethereum until the fees reduce. That
+              is where layer 2 comes in to scale Ethereum today.
+            </p>
+          </FlexContainer>
+        </TwoColumnContent>
+        <h3>Benefits</h3>
+        <InfoGrid>
+          {layer2Cards.map(({ emoji, title, description }, idx) => (
+            <Card
+              description={description}
+              title={title}
+              emoji={emoji}
+              key={idx}
+            />
+          ))}
+        </InfoGrid>
+      </PaddedContent>
+
+      <PaddedContent>
+        <TwoColumnContent>
+          <FlexContainer flexPercent="50">
+            <h2>How layer 2 works</h2>
+            <p>
+              As we mentioned above, Layer 2 is a collective term for Ethereum
+              scaling solutions that handle transactions off Ethereum layer 1
+              while taking advantage of the robust decentralized security model
+              of Ethereum layer 1.{" "}
+              <b>
+                A layer 2 is a separate blockchain that is connected to Ethereum
+              </b>
+              . How does that work?
+            </p>
+            <p>
+              A layer 2 blockchain regularly communicates with Ethereum (by
+              submitting bundles of transactions) in order to ensure it has
+              similar security and decentralization guarantees, and requires no
+              changes to the layer 1 protocol (Ethereum). This lets layer 1
+              handle security, data availability, and decentralization, while
+              the layer above (layer 2) handles scaling. Layer 2s takes the
+              transactional burden away from the layer 1, and posts finalized
+              proofs back to the layer 1 to finalize the state. By taking this
+              transaction load away from layer 1, the base layer will become
+              less congested, and everything becomes more scalable.
+            </p>
```

Comment:
```suggestion
            <p>
              Layer 2 lets Ethereum do what it does best: handling security, guaranteeing that data is available, and ensuring the network has high decentralization. While Ethereum is busy handling this, layer 2 handles scaling. It does this by taking on the burden of processing individual transactions, then sending proof back to Ethereum that this processing work was completed.
            </p>
            <p>
              Because layer 2 regularly communicates with Ethereum, submitting bundles of transactions, it ensures layer 2 has a similar level of security and decentralization guarantees to its parent. And, taking this transaction load away from Ethereum frees up space, letting everything becomes more scalable.
            </p>
```

Reply:
Noticed this was out of date but not added @corwintines, LMK what you think.

Goes along the lines of what @samajammin was saying about layer 1 === Ethereum. I feel like in some places, trying to explain things in layer 1, layer 2 terms becomes a bit of a word salad.